server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -360,7 +361,7 @@ module.exports = {
         title: 'Postpone or refuse',
         label: postponement.getLabel({ decisions }),
         action: postponement.getAction({ decisions }),
-        visible: validAddress,
+        visible: validAddress && dmRefused !== true,
```

Comment:
Can this just be `visible: validAddress && !dmRefused`

Reply:
yep